molgenis-api-metadata/src/main/java/org/molgenis/api/metadata/v3/MetadataApiServiceImpl.java

```diff
@@ -29,38 +35,38 @@
   private final SortMapper sortMapper;
   // TODO replace usage of DataService with new methods in MetaDataService
   private final DataService dataService;
+  private final MetadataApiJobService metadataApiJobService;
 
   MetadataApiServiceImpl(
       MetaDataService metadataService,
       QueryMapper queryMapperV3,
       SortMapper sortMapper,
-      DataService dataService) {
+      DataService dataService,
```

Comment:
Undesirable dependency, can we use ```MetadataService``` only? Optionally add a todo to the metadata API PR to tackle separately.

Reply:
I've added an item to the todo list in the API PR.